components/configuration-accessor/default/package.mjs

```diff
@@ -16,51 +20,104 @@ const isPrefixArray = (prefix, array) => {
   }
 };
 
-export const generateParseSource = (name) => {
-  const regexps = [
-    new RegExp(`^(?<before>${name})(?<after>($|\\s[\\s\\S]*$))`, "u"),
-    new RegExp(`^(?<before>npx\\s+${name})(?<after>($|\\s[\\s\\S]*$))`, "u"),
-    new RegExp(
-      `^(?<before>npm\\s+exec\\s+mocha)(?<after>($|\\s[\\s\\S]*$))`,
-      "u",
+const findPositionalIndex = (tokens, index) => {
```

Comment:
This function could use a comment and perhaps a rename to better describe what it's designed to find. Would `findScriptName` be more accurate?

Reply:
I agree that `positional` was misleading. I used `findScriptIndex` 57810965cfb89876fd7e00b4a75b5783facdb784